crates/snp_validation/src/lib.rs

```diff
@@ -367,9 +377,111 @@ impl Snp {
             }
         }
 
+        if config.is_some() {
+            let config_ref = config.as_ref().unwrap();
+
+            if config_ref.minimum_abi.is_some() {
+                let req_ver = config_ref.minimum_abi.as_ref().unwrap();
+                let req = VersionReq::parse(&format!(">={}", &req_ver)).unwrap();
+                if !req.matches(&report.body.policy.into()) {
+                    return Err(anyhow!("snp minimum abi not met"));
+                }
```

Comment:
use `ensure!` https://docs.rs/anyhow/latest/anyhow/macro.ensure.html

Reply:
I don't see how `ensure!` works here, since I'm not checking that two values are equal.

Reply:
I did `ensure!` some other areas where it made more sense to me.